src/twisted/names/test/test_names.py

```diff
@@ -1221,13 +1223,50 @@ def loadBindString(self, s):
         @return: a new bind authority
         @rtype: L{twisted.names.authority.BindAuthority}
         """
-        fp = FilePath(self.mktemp().encode("ascii"))
+        fp: FilePath[AnyStr] = FilePath(path)
```

Comment:
Does mypy complain without this annotation? I would imagine `FilePath[AnyStr]` would be implied by `FilePath(<something: AnyStr>)`. I'd leave it to the inference unless it is required.
```suggestion
        fp = FilePath(path)
```

Reply:
It does not.